src/Microsoft.AspNet.Http.Abstractions/Authentication/AuthenticationManager.cs

```diff
@@ -34,7 +39,7 @@ public virtual Task ChallengeAsync()
 
         public virtual Task ChallengeAsync(AuthenticationProperties properties)
         {
-            return ChallengeAsync(authenticationScheme: string.Empty, properties: properties);
+            return ChallengeAsync(authenticationScheme: AutomaticScheme, properties: properties);
```

Comment:
If we're going to do this should we also change the null checks to IsNullOrEmpty?